_posts/2016-03-24-app-analytics-redux.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "App Analytics with Redux"
+date: 2016-03-24 11:25:00
+categories: analytics redux segment data monitoring
+author: josh.habdas
+cover: 
+description: Learn how to track app analytics with Redux the easy way using Segment.
+comments: true
+disqus: true
+---
+
+So you figured out [Where Flux Went Wrong] and are shipping your app with Redux. How will you measure usage? Will you know how users are using it once it's launched? What about user authentication? You'll definitely want to track that. But how will you do it?
+
+Will you add a Google Analytics integration? Perhaps you're planning to rock your own dashboards with [Keen.io]. But what about heat maps, page read statistics, e-commerce integration, or the ability to watch video recordings of your users? Which tools will you choose? How many integrations will you need? How well will those integrations work with Redux?
+
+If you're asking these questions it's a good thing. But the analytics tooling space is vast. With so many options which will you choose?
+
+I'm going to let you in on a little tip: don't choose any of them, choose as many as you can.
+
+> In this post you will learn how to enhance your reducers to connect your app to hundreds of analytics tooling integration a single dependency, at very little cost and with almost no effort.
+
+## Analytics is about learning
+
+Have you ever heard of Segment? If not, let me give you the skinny.
+
+[Segment], the makers of is a analytics aggregation service providing [hundreds of analytics tooling integrations](https://segment.com/integrations) with the toggle of a switch. It's reliable and easy to work with. So easy, in fact, I [added it to my blog](https://github.com/jhabdas/habd.as#features). Their pricing model is moderate, even at scale. And, best of all, you can try out dozens of popular integrations such as Google Analytics for free, without any development experience needed.
```

Comment:
Was going to say the makers of NightmareJS but it felt out of scope. Thanks for catching the flub.